files.json

```diff
@@ -491,6 +495,18 @@
     "SHA": "3edc22f8332231c3ba8be246f184b736b8d28f06ce24f08168d8ecf052549769",
     "URL": "https://storage.googleapis.com/golang/go1.16.1.linux-amd64.tar.gz"
   },
+  "go1.16.2.linux-amd64.tar.gz": {
+    "SHA": "542e936b19542e62679766194364f45141fde55169db2d8d01046555ca9eb4b8",
+    "URL": "https://dl.google.com/go/go1.16.2.linux-amd64.tar.gz"
+  },
+  "go1.16.6.linux-amd64.tar.gz": {
```

Comment:
Should we add `go1.16.{3,4,5}` too?

Reply:
Oddly, 1.16.{2,6,7} were the ones detected to be missing by the sync script. Will try the others.

Reply:
Added in 52dc633e0bdc1c9fdfbceddd81734156154a4e5a